### ISSUE TITLE
JupyterLab doesn't build on Onyx

### ISSUE THREAD

**Issue by nchaimov** [ParaToolsInc/taucmdr#242]
JupyterLab fails to build on Onyx due to what looks like a missing dependency that we're not installing correctly.

```
> npm run build

> @jupyterlab/application-top@0.10.2 build /p/home/nchaimov/taucmdr-inst/anaconda-4.4.0/share/jupyter/lab/staging
> webpack

Hash: ad9d94f79c9895b0224f
Version: webpack 2.7.0
Time: 8058ms
                                 Asset     Size  Chunks                    Chunk Names
  674f50d287a8c48dc19ba404d20fe713.eot   166 kB          [emitted]
af7ae505a9eed503f8b8e6982036873e.woff2  77.2 kB          [emitted]
 fee66e712a8a08eef5805a46892932ad.woff    98 kB          [emitted]
  b06871f281fee6b241d60582ae9369b9.ttf   166 kB          [emitted]
  912ec66d7572ff821749319396470bde.svg   444 kB          [emitted]  [big]
                           0.bundle.js   894 kB       0  [emitted]  [big]
                        main.bundle.js  7.83 MB       1  [emitted]  [big]  main
                       0.bundle.js.map  1.08 MB       0  [emitted]
                    main.bundle.js.map  9.42 MB       1  [emitted]         main
  [27] ./~/@jupyterlab/application/lib/index.js 5.48 kB {1} [built]
 [468] ./~/@jupyter-widgets/jupyterlab-manager/lib/index.js 346 bytes {1} [optional] [built]
 [469] ./~/@jupyterlab/application-extension/lib/index.js 6.12 kB {1} [optional] [built]
 [470] ./~/@jupyterlab/apputils-extension/lib/index.js 7.91 kB {1} [optional] [built]
 [471] ./~/@jupyterlab/codemirror-extension/lib/index.js 10.5 kB {1} [optional] [built]
 [472] ./~/@jupyterlab/completer-extension/lib/index.js 7.09 kB {1} [optional] [built]
 [473] ./~/@jupyterlab/console-extension/lib/index.js 12.5 kB {1} [optional] [built]
 [474] ./~/@jupyterlab/csvviewer-extension/lib/index.js 1.79 kB {1} [optional] [built]
 [475] ./~/@jupyterlab/docmanager-extension/lib/index.js 10.1 kB {1} [optional] [built]
 [476] ./~/@jupyterlab/faq-extension/lib/index.js 3.78 kB {1} [optional] [built]
 [477] ./~/@jupyterlab/filebrowser-extension/lib/index.js 11.9 kB {1} [optional] [built]
 [478] ./~/@jupyterlab/fileeditor-extension/lib/index.js 11.8 kB {1} [optional] [built]
 [479] ./~/@jupyterlab/help-extension/lib/index.js 8.84 kB {1} [optional] [built]
 [480] ./~/@jupyterlab/imageviewer-extension/lib/index.js 4.25 kB {1} [optional] [built]
 [510] ./build/index.out.js 8.38 kB {1} [built]
    + 1271 hidden modules
/p/home/nchaimov/taucmdr-inst/anaconda-4.4.0/bin/jupyter labextension list 2>&1 | grep -q jupyterlab_bokeh || (cd build/jupyterlab/jupyterlab_bokeh && /p/home/nchaimov/taucmdr-inst/anaconda-4.4.0/bin/npm install && /p/home/nchaimov/taucmdr-inst/anaconda-4.4.0/bin/jupyter labextension link --no-build . )
/usr/bin/env: node: No such file or directory
Makefile:221: recipe for target 'jupyterlab-extensions-install' failed
make: *** [jupyterlab-extensions-install] Error 127
```

**Comment by jlinford** [comment]
Ditto on east03

**Comment by zbeekman** [comment]
Temporary work around:

```patch
diff --git a/Makefile b/Makefile
index 85ccd62..acd1fb1 100644
--- a/Makefile
+++ b/Makefile
@@ -190,7 +190,7 @@ install: build
        @echo "-------------------------------------------------------------------------------"
        @echo

-python_check: $(PYTHON_EXE) jupyterlab-install jupyterlab-extensions-install
+python_check: $(PYTHON_EXE) # jupyterlab-install jupyterlab-extensions-install
        @$(PYTHON) -c "import sys; import setuptools;" || (echo "ERROR: setuptools is required." && false)

 python_download: $(CONDA_SRC)
```

**Comment by nchaimov** [comment]
Fixed in 1c3495933d1e0e22d3b5d7c129cf48fa56f5ddcc
